CHANGELOG.md

```diff
@@ -1,21 +1,27 @@
 ## master / unreleased
 
+## 0.2.0-rc.0 / 2019-08-28
+
+This release has several exciting features, the most notable of them being setting `-ingester.spread-flushes` to potentially reduce your chunk storage space by upto 66%.
```

Comment:
Quick thought: it's 66% less chunk space but the index space is only a little less.
Maybe the headline should be "reduce storage by up to 50%" ?